commit 78f710dc186f34fb14d8b22a33749a56013e7b85
tree 6e750e0ef0022c95bd0b27b8625e9873ce31a5ff
parent b3bd6fe96da32e4e818b51826f04349cee9b15ba
author Masakazu Mokuno <mokuno@sm.sony.co.jp> 1184920139 +0900
committer Jeff Garzik <jeff@garzik.org> 1185308919 -0400

    ps3: removed conditional ethtool support
    
    Removed conditional ethtool support.  Always enabled.
    
    Signed-off-by: Masakazu Mokuno <mokuno@sm.sony.co.jp>
    Signed-off-by: Jeff Garzik <jeff@garzik.org>

---
 drivers/net/gelic_net.c |    4 ----
 1 file changed, 4 deletions(-)

--- a/drivers/net/gelic_net.c	2007-08-27 14:01:21.000000000 -0400
+++ b/drivers/net/gelic_net.c	2007-08-27 14:03:57.000000000 -0400
@@ -1162,7 +1162,6 @@ alloc_tx_failed:
 	return -ENOMEM;
 }
 
-#ifdef GELIC_NET_ETHTOOL
 static void gelic_net_get_drvinfo (struct net_device *netdev,
 				   struct ethtool_drvinfo *info)
 {
@@ -1282,7 +1281,6 @@ static struct ethtool_ops gelic_net_etht
 	.get_rx_csum	= gelic_net_get_rx_csum,
 	.set_rx_csum	= gelic_net_set_rx_csum,
 };
-#endif
 
 /**
  * gelic_net_tx_timeout_task - task scheduled by the watchdog timeout
@@ -1350,9 +1348,7 @@ static void gelic_net_setup_netdev_ops(s
 	/* NAPI */
 	netdev->poll = &gelic_net_poll;
 	netdev->weight = GELIC_NET_NAPI_WEIGHT;
-#ifdef GELIC_NET_ETHTOOL
 	netdev->ethtool_ops = &gelic_net_ethtool_ops;
-#endif
 }
 
 /**
